src/directive/rateLimit.ts

```diff
@@ -0,0 +1,33 @@
+import {
```

Comment:
Inspiration came from one of their examples: https://github.com/ravangen/graphql-rate-limit/blob/main/examples/context/index.js